pricing/v1/pricing.proto

```diff
@@ -46,23 +46,34 @@ message GetInfoResponse {
 }
 
 // Request message for GetPricingInfo
-message GetPricingInfoRequest {
+message GetPricingRequest {
+
+  //Required cloud (AWS/Azure)
   string cloud = 1;
+
+  // Required service. 
```

Comment:
Are these strings the actual strings that the user can use as valid inputs? I think it would be helpful to give users a reference as to what actual strings are valid inputs.

Reply:
Also, put a period after the Required/Optional words. Like so,
`Required. Cloud vendor service.`